Libraries/LibURL/URL.cpp

```diff
@@ -248,6 +248,20 @@ String URL::serialize_path() const
     return output.to_string_without_validation();
 }
 
+// This function is used whenever a path is needed to access the actual file on disk.
+// On Windows serialize_path can produce a path like /C:/path/to/tst.htm, so the leading slash needs to be removed to obtain a valid path.
+ByteString URL::file_path() const
+{
+    ByteString path = percent_decode(serialize_path());
+
+#ifdef AK_OS_WINDOWS
+    if (path[0] == '/')
```

Comment:
```suggestion
    if (path.starts_with('/'))
```

Otherwise I believe this would crash if `file_path()` is called on it
```
new URL('thing:', 'resource://some/resource/')
```

Reply:
Thanks. Turns out I made this mistake [before][1].

[1]: https://github.com/LadybirdBrowser/ladybird/blob/105096e75a/AK/LexicalPathWindows.cpp#L63